soaktest/soaktest.go

```diff
@@ -251,7 +251,8 @@ func HandleMessage(e *kafka.Message,
 // lost messages from consumer side
 func verifyMessage(e *kafka.Message,
 	hwmarks map[string]uint64, msgid uint64) {
-	tags := []string{fmt.Sprintf("topic:%s", *e.TopicPartition.Topic)}
+	tags := []string{fmt.Sprintf("topic:%s partition:%d",
```

Comment:
Is this how multiple tags are passed? Seeing how tags is an array of strings shouldnt it be one key:val per array item?
(I don't know, it just looks a bit uncommon)